modules/gdscript/gdscript_warning.cpp

```diff
@@ -162,6 +162,12 @@ String GDScriptWarning::get_message() const {
 			return vformat(R"*(The default value is using "%s" which won't return nodes in the scene tree before "_ready()" is called. Use the "@onready" annotation to solve this.)*", symbols[0]);
 		case ONREADY_WITH_EXPORT:
 			return R"("@onready" will set the default value after "@export" takes effect and will override it.)";
+		case ACCESS_PRIVATE_MEMBER:
+			CHECK_SYMBOLS(1);
+			return vformat(R"(Trying to access a private member "%s" from an external place, which would cause problems during runtime.)", symbols[0]);
```

Comment:
I'm not sure what _exactly_ to say here, but this warning is not always accurate. It's not that it "would cause problems". Rather, it's not intended and it _may_ cause problems.
Also we generally refer to "runtime" as when the _project_ is run, but this warning may also be sent for `tool` script meant to run in the editor.
```suggestion
			return vformat(R"(The member "%s" is private. It is not meant to be accessed from an external script.)", symbols[0]);
```
or
```suggestion
			return vformat(R"(The member "%s" is private. It should not be accessed from an external script.)", symbols[0]);
```
or, the opposite:
```suggestion
			return vformat(R"(The member "%s" is private. It should only be accessed by a script that inherits this member.)", symbols[0]);
```
Or similar? Or not at all? I'd like others to chime in as well.